packages/sample-app/sample_app/weaviate_v3.py

```diff
@@ -1,40 +1,37 @@
 # Tested with weaviate-client==3.26.0
+# Weaviate instrumentation with opentelemetry-instrumentation-weaviate==0.19.0
```

Comment:
this can be removed, right? cause we support both

Reply:
Are there any cassettes left? I removed both "fetch" cassettes, the rest is needed afais.